suite-common/walletconnect/src/walletConnectThunks.ts

```
@@ -282,14 +274,17 @@ export const updateAccountsThunk = createThunk(
             const updatedNamespaces = getNamespaces(accounts);
-            const namespaces = {
-                ...oldNamespaces,
-                eip155: {
-                    ...oldNamespaces.eip155,
-                    accounts: updatedNamespaces.eip155.accounts,
-                    chains: updatedNamespaces.eip155.chains,
-                },
-            };
+            const namespaces = Object.keys(oldNamespaces).reduce(
+                (acc, key) => ({
+                    ...acc,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)